doc-content/jbpm-docs/src/main/asciidoc/BPMN2/create-service-task-proc.adoc

```diff
@@ -1,38 +1,68 @@
+. Click *Upload* to open the *Artifact upload* window.
+. Choose the `.jar` file and click image:upload-button.png[].
++
+[IMPORTANT]
+====
+The `.jar` file contains data types (data objects) and Java classes for web service and Java service tasks respectively.
+====
+. Create a project you want to use.
+. Go to your project *Settings -> Dependencies*.
+. Click *Add from repository*, locate the uploaded `.jar` file, and click *Select*.
+. Save the project.
++
+--
+.Example of creating web service task
+The default implementation of a service task in the BPMN2 specification is a web service. The web service support is based on the Apache CXF dynamic client, which provides a dedicated service task handler that implements the `WorkItemHandler` interface:
 
+`org.jbpm.process.workitem.bpmn2.ServiceTaskHandler`
 
-.Procedure
-. Create a business process.
-. In the process designer, select the *Activities* tool from the tool palette.
-. Select *Service Task*.
-. Click a blank area of the process designer canvas.
-. If necessary, in the upper-right corner of the screen, click the *Properties* icon.
-. Add or define the task information listed in the following table as required.
-+
-.Service task parameters
+To create a service task using web service, you must configure the web service:
+
+.. Create a business process.
+.. If necessary, in the upper-right corner of the screen, click the *Properties* icon.
+.. Click image:import-property-icon.png[] in the *Imports* property to open the *Imports* window.
+.. Click *+Add* next to the *WSDL Imports* to import the required WSDL (Web Services Description Language) values. For example:
++
+* *Location*: http://localhost:8080/sample-ws-1/SimpleService?wsdl
++
+The location points to the WSDL file of your service.
++
+* *Namespace*: http://bpmn2.workitem.process.jbpm.org/
++
+The namespace must match `targetNamespace` from your WSDL file.
+
+.. In the process designer, select the *Activities* tool from the tool palette.
+.. Select *Service Task*.
+.. Click a blank area of the process designer canvas.
+.. Add or define the task information listed in the following table as required.
++
+.Web service task parameters
 [cols="30%,70%", options="header"]
 |===
 |Label
 |Description
 
 | *Name*
-| The name of the service task.
+| The name of the service task. You can also double-click the shape to edit the name of the service task.
 
 | *Documentation*
 | Enter a description of the task. The text in this field is included in the process documentation. Click the *Documentation* tab in the upper-left side of the process designer canvas to view the process documentation.
 
 | *Implementation*
-| Specify whether the task is implemented in Java or is a web service
+| Specify a web service.
 
 | *Interface*
-| The class used to implement the script, for example `org.xyz.HelloWorld`.
+| The class used to implement the script, such as `CountriesPortService`.
```

Comment:
This is not a class, just a name of the service. There are no classes in case of a WebService.

Reply:
Added 
The service used to implement the script, such as `CountriesPortService`.